### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by DennisPintilie**
- installed eslint and prettier
- configured eslint and prettier
- fixed leftover linting errors

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -14,10 +14,12 @@ app.use(express.static(CLIENT_PATH));
 
 app.post('/createProfile', (req, res) => {
   console.log("hit", req.body);
```
Comment (jstrzesz) [inline server/index.js]: remove console.log

File: server/index.js [hunk 2]
```diff
@@ -14,10 +14,12 @@ app.use(express.static(CLIENT_PATH));
...
+  res.sendStatus(201);
 })
 
 app.put('/updateProfile', (req, res) => {
   console.log('updateProfile hit', req.body);
```
Comment (jstrzesz) [inline server/index.js]: remove console.log

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "node": true,
+    "es6": true
+  },
+  "plugins": ["import", "require"],
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
+  "rules": {
+    "no-console": "warn",
```
Comment (jstrzesz) [inline .eslintrc]: change warn to error

File: package.json [hunk 4]
```diff
@@ -32,23 +35,27 @@
...
+    "husky": "^4.2.5",
+    "prettier": "^2.1.1",
     "webpack": "^4.44.1",
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "echo \"pre-commit: Pre-commit working...\"",
+      "pre-commit": "npm run lint:fix",
```
Comment (jstrzesz) [inline package.json]: include npm run format

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

File: .eslintrc (hunk 3 above)
Comment (DennisPintilie) [reply .eslintrc]: If I change to error, then we would have to delete our console logs that let us know if the server starts and if the database connects successfully.

File: package.json (hunk 4 above)
Comment (DennisPintilie) [reply package.json]: Running npm run format will not fix any errors at all.

File: .eslintrc (hunk 3 above)
Comment (jstrzesz) [reply .eslintrc]: replace those specific console.logs with console.info

Comment (DennisPintilie) [reply .eslintrc]: still getting a console error:  error  Unexpected console statement -- no-console

**Review by bsbrionesOS — APPROVED** [review]
Approved!

**Review by Evtaft22 — APPROVED** [review]

**Pull request merged**